app/models/requests/request.rb

```diff
@@ -159,9 +158,8 @@ def load_items
     end
 
     def thesis?
-      unless doc[:holdings_1display].nil?
-        return true if parse_json(doc[:holdings_1display]).key?('thesis')
-      end
+      return if doc[:holdings_1display].nil?
+      return true if parse_json(doc[:holdings_1display]).key?('thesis')
```

Comment:
can be just ` parse_json(doc[:holdings_1display]).key?('thesis')` which will return true if it is true and false if not.  Unless there is a test for nil?  it is a nicer form.  The whole method could also be all one line `doc[:holdings_1display].present? && parse_json(doc[:holdings_1display]).key?('thesis')`

Reply:
updated